### PR TITLE
DOCS-2289: Advanced Helm workflow

### PR DESCRIPTION
**Pull request by ctauchen**

### CODE REVIEW

File: calico-cloud/get-started/install-automated.mdx
Comment (ctauchen) [inline calico-cloud/get-started/install-automated.mdx:1]: @tmjd The rough outline for you.

Main changes:
ADDED § About customizing your Helm installation
ADDED § Prepare your values.yaml with customizations
MODIFIED § Install Calico Cloud as part of an automated workflow

Lots of glue and filler yet to be applied, but this represents the structure and workflow.

No formal review needed, but if you have thoughts, would be happy to hear.

CC @jaderhs

File: calico-cloud/get-started/install-automated.mdx [hunk 1]
```diff
@@ -39,6 +42,110 @@ When the key is deleted, all API requests based on that key will be rejected.
 
 :::
 
+## About customizing your Helm installation
+
+<!-- As a user, I want to know what I can do with  so I can  -->
+
+You can customize your Calico CLoud installation for the following purposes:
```
Comment (jaderhs) [inline calico-cloud/get-started/install-automated.mdx]: Uppercase L typo in `CLoud`

File: calico-cloud/get-started/install-automated.mdx
Comment (tmjd) [reply calico-cloud/get-started/install-automated.mdx:1]: This doc seems to conflate the automated install with the customization. The customization is still possible when using the non-automated helm install (like using the helm command retrieved from the UI.

File: calico-cloud/get-started/install-automated.mdx [hunk 2]
```diff
@@ -47,69 +154,36 @@ These commands can be added to any automated workflow.
...
-   helm repo add calico-cloud https://installer.calicocloud.io/charts --force-update && \
    helm upgrade --install calico-cloud-crds calico-cloud/calico-cloud-crds \
    --namespace calico-cloud \
    --create-namespace
    ```
 
-   </TabItem>
-   </Tabs>
-
 1. Apply the client credentials secret to your cluster.
```
Comment (tmjd) [inline calico-cloud/get-started/install-automated.mdx:174]: This step is only part of the "automated install", so it kind of seems like customization is only available with the "automated install".

File: calico-cloud/get-started/install-automated.mdx [hunk 3]
```diff
@@ -39,6 +42,110 @@ When the key is deleted, all API requests based on that key will be rejected.
...
+:::note
+
+If you're upgrading from an older version of Calico Cloud, the Container Threat Detection and Packet Capture features will remain enabled unless you explicitly set them to `Disabled`.
+
+:::
+
+### Optional parameters for pod scheduling and resource management
+
+For many Calico Cloud components, you can specify node selectors, tolerations, and resource requests and limits.
+The full list of Calico Cloud components is available in the default `values.yaml` file.
```
Comment (tmjd) [inline calico-cloud/get-started/install-automated.mdx:100]: Do we only want to provide this info in the values.yaml file? There is plenty of reference documentation we could link to by including things like
`installer.components.apiServer` and linking to the reference/installation/api#operator.tigera.io/v1.APIServerDeployment

File: calico-cloud/get-started/install-automated.mdx
Comment (tmjd) [reply calico-cloud/get-started/install-automated.mdx:1]: I chatted with Chris, this is purposeful to not document the customization (tolerations/requests/limits/...) for non-automated installs.

**Review by tmjd — APPROVED** [review]

**Pull request merged**